### PR TITLE
[test] Add test case for tanh and maximum op

### PR DESCRIPTION
**Pull request by cuiyanx**
Signed-off-by: cuiyan <yanx.cui@intel.com>

### CODE REVIEW

**Comment by cuiyanx** [comment]
@BruceDai please review

**Comment by cuiyanx** [comment]
Fix: #610 and #580

File: test/unit-test/test-Model.js [hunk 1]
```diff
@@ -6493,6 +6493,123 @@ describe('Unit Test/Model Test', function() {
...
+        let output = {type: nn.FLOAT32};
+        model.addOperand(input);
+        model.addOperand(output);
+        assert.throws(() => {
+          model.addOperation(nn.TANH, [0], [1]);
+        });
+      });
+    });
+
+    it('"4-D tensor as two inputs and one output, the type of input and output being TENSOR_FLOAT32 type" are ok for "MAXIMUM" operation', function() {
```
Comment (BruceDai) [inline test/unit-test/test-Model.js]: @cuiyanx Please add more test case because MAXIMUM op does also support [TENSOR_INT32 and TENSOR_QUANT8_ASYMM type](
https://github.com/intel/webml-polyfill/blob/master/src/nn/Enums.js#L1235)

File: test/cts/test/cts-all.js [hunk 2]
```diff
@@ -2804,183 +2804,6 @@ describe('CTS', function() {
     }
   });
 
-  it('check result for Conv float relu example', async function() {
```
[the remark this answers is not in the document]
Comment (cuiyanx) [reply test/cts/test/cts-all.js]: I will update cts tool for later.

**Comment by cuiyanx** [comment]
@BruceDai please review again

**Comment by cuiyanx** [comment]
Add 30 test cases for "TANH" and "MAXIMUM" op:
               CTS:   "TANH" op: 1 test case
             PLUS:   "MAXIMUM" op: 3 test cases
  test-Model.js:   "TANH" op: 12 test case    "MAXIMUM" op: 14 test cases

**Comment by cuiyanx** [comment]
@BruceDai please review again
